frontend/src/components/ProjectServiceAccountRow.vue

```diff
@@ -129,41 +122,11 @@ export default {
   name: 'project-service-account-row',
   components: {
     TimeString,
-    GPopper,
     AccountAvatar,
-    ServiceAccountRoles
+    AccountRoles
   },
   props: {
-    username: {
-      type: String,
-      required: true
-    },
-    avatarUrl: {
-      type: String,
-      required: true
-    },
-    displayName: {
-      type: String,
-      required: true
-    },
-    createdBy: {
-      type: String
-    },
-    description: {
-      type: String
-    },
-    creationTimestamp: {
-      type: String
-    },
-    created: {
-      type: String
-    },
-    roles: {
-      type: Array,
-      required: true
-    },
-    roleDisplayNames: {
-      type: Array,
+    item: {
```

Comment:
Why do we change the explicit behavior to an implicit untyped?

Reply:
@petersutter refactored the code and added this component for reuse in the terminal feature context. However, he missed to re-use the component in the user context but it is the very same code so re-use here makes 100% sense. However, the name `ServiceAccount` makes no sense if we use it ofr all account types (User and Service Accounts)